### DISCUSSION TITLE
ExceptionMessage : SQLSTATE[HY000] [2002] Operation now in progress on Laravel

### DISCUSSION THREAD

**Discussion by shiratsu** [laravel/framework#32911] [General]
hi.

this error happened suddenly during this SQL on production environment.
i don know why this error happened.

```
/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Connection.php
ExceptionLine : 664
ExceptionMessage : SQLSTATE[HY000] [2002] Operation now in progress (SQL: select `works`.*, `client_users`.`client_name`, `client_users`.`bridge_company_id`, `client_users`.`name` as `client_user_name`, `client_users`.`email` as `client_user_email`, `client_users`.`phone` as `client_user_phone` from `works` inner join `client_users` on `works`.`client_user_id` = `client_users`.`client_user_id` where `works`.`work_id` = WOxxxxxxxxxx and `works`.`work_status` = 02 limit 1)
```

it's not on migration.
it usually works fine.

it happened midnight.
it means there are less access than afternoon.
so i don know why it happens.
there are no error and slow query at Amazon Aurora mysql.
work_id is a primary key for this table,so i don think this sql is slow.

Anyone with ideas or knowledges regarding to this problem?

our laravel version is 5.8.23.
ec2 server is t3.medium.
aurora is t3.medium.
aurora has an write instance and a read instance.

**Comment by mreduar** [comment]
Were you able to find a solution to your problem?

**Comment by ocind** [comment]
hi @shiratsu did you find a solution to this problem? We are currently having the same issue
